protocol/x/revshare/keeper/revshare.go

```diff
@@ -117,3 +118,22 @@ func (k Keeper) SetUnconditionalRevShareConfigParams(ctx sdk.Context, config typ
 	unconditionalRevShareConfigBytes := k.cdc.MustMarshal(&config)
 	store.Set([]byte(types.UnconditionalRevShareConfigKey), unconditionalRevShareConfigBytes)
 }
+
+func (k Keeper) ValidateRevShareSafety(
```

Comment:
Thanks for adding this! Could we add comment for what we are checking (in math formula)